pages/02.Tutorials/09.dnsaas/docs.en.md

```diff
@@ -0,0 +1,390 @@
+---
+title: 'Using DNS as a service'
+date: '24-07-2019 12:00'
+taxonomy:
+    category:
+        - docs
+---
+
+## Using DNS as a Service (DNSaaS)
```

Comment:
IMO a link to the tutorial and how-to guide is missing from the reference guide page. I think it is nice for the user when reading the reference guide to find respective tutorials and how tos.